attributes/default.rb

```diff
@@ -0,0 +1,12 @@
+default['luigi']['user'] = 'luigi'
+default['luigi']['group'] = 'luigi'
+default['luigi']['dir'] = '/etc/luigi'
+
+default['luigid']['user'] = 'luigi'
+default['luigid']['group'] = 'luigi'
+default['luigid']['dir'] = '/mnt/luigid'
```

Comment:
This structure sounds strange. It is not clear what goes inside `default['luigid']['dir']`